examples/nvidia-vgpu-device-manager-mig-support-example.yaml

```diff
@@ -0,0 +1,119 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: vgpu-device-manager
+  namespace: default
+  labels:
+    app: vgpu-device-manager
+spec:
+  selector:
+    matchLabels:
+      app: vgpu-device-manager
+  template:
+    metadata:
+      labels:
+        app: vgpu-device-manager
+    spec:
+      serviceAccount: vgpu-device-manager
+      serviceAccountName: vgpu-device-manager
+      containers:
+      - name: vgpu-device-manager
+        image: nvcr.io/nvidia/cloud-native/vgpu-device-manager:v0.1.0
```

Comment:
nit: let's update this to the latest release tag `v0.3.0`

Reply:
Absolutely, I have updated this one and feel free to also check [this](https://github.com/NVIDIA/vgpu-device-manager/pull/93#discussion_r2163493586) comment regarding the original example YAML manifest.